source/quick-start.txt

```diff
@@ -7,193 +7,182 @@ Quick Start
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-This guide shows you how to create an application that uses the Node.js
-driver to connect to a MongoDB Atlas cluster.
+*Estimated time to complete this guide: 35 minutes.*
 
-Check for Node and NPM
-----------------------
+This guide shows you how to create an application that uses the
+**Node.js driver** to connect to a **MongoDB Atlas cluster**.
 
-Ensure your system has Node version 8 or later and NPM (Node Package
-Manager) installed. Run the following commands to check that you meet
-these requirements:
+The Node.js driver is an interface through which you can connect to and
+communicate with MongoDB instances. If you prefer to connect to MongoDB using
+a different programming language, select from the ones on our
```

Comment:
We say here that users can connect with a "programming language" but in the previous sentence we say they can connect using an "interface". Let's try to be consistent with our terminology, as we already use "driver" and "interface".

Reply:
Ok, I think to address this, I can just change "ones" to "drivers".

Reply:
My feedback was centered around the possible confusion around "connecting **with** a programming language" – they're connecting with a tool written in a language they prefer, vs. just the programming language (which to me, is a bit confusing – "Connect to your cluster with Python" wouldn't make too much sense to me).

I meant something like:

"If you prefer to connect to MongoDB using a driver [for/with] a different programming language, please select a different driver listed on our..."

We'd select either "for" or "with" in the example sentence above ^

Reply:
1)  I think the correction I identified needs to exist to distinguish between "programming language" and "driver".

2) Can we punt on "with" vs "using? I'm not sure I see any advantage to changing it since we use those interchangeably across our docs. I prefer the latter because it implies the object is a requirement whereas "with" does not have as strong an association and is used more to describe "accompanied by" but does not imply the object is required.

Reply:
1. I'm lost now.

2. Sure.